homeassistant/components/openai_conversation/conversation.py

```diff
@@ -131,10 +139,19 @@ async def _transform_stream(
                         )
                     ]
                 }
+            elif choice.finish_reason == "length":
+                if chunk.id == last_id_with_content:
+                    LOGGER.warning("Message truncated because max tokens reached")
+                    yield {"content": "..."}
```

Comment:
The frontend uses `…` to indicate more text is coming so this can be confusing. Should we just inject an error? `>> Error: response too long`

Reply:
Yes, that's why I think it's a good indicator of an unfinished text. The user will notice that no more text is coming because it usually comes fast.
Also an error text in English might produce interesting results on non-English TTS.

Or you mean I need to change `...` into `…`?

Reply:
I gave it another thought, and now I think we should just raise an error.
At this point we already sent to chat_log all available content, and it is up to frontend or TTS how to properly render and localize it.